git-gone

```diff
@@ -82,9 +82,9 @@ export LC_MESSAGES=C
 gone="$BRANCH/.*: gone]"
 if [ $delete -eq 1 ]; then
   if [ $force -eq 1 ]; then
-    git branch -vv | grep "$gone" | awk '{print $1}' | xargs --no-run-if-empty git branch -D
+    git branch -vv | grep "$gone" | awk '{print $1}' | xargs git branch -D
```

Comment:
I'd be happy to merge this to get the script working again on macOS. I use macOS too but I've been using an older version so I didn't realize the bug.

This was added in #1 by @sergiuszkierat I guess to address some issue he was seeing on Linux. I wonder if we should reverse engineer a bug report so we can address it without breaking mac users?

Reply:
Unfortunately, I can't find why @sergiuszkierat added `--no-run-if-empty` option. I think he wanna improve this script, I guess :)

We can add OS check condition in this script.